src/components/overflow-menu-drawer/CollectionOverflowMenuDrawer.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import {
+  FavoriteSource,
+  RepostSource,
+  ShareSource
+} from 'audius-client/src/common/models/Analytics'
+import { ID } from 'audius-client/src/common/models/Identifiers'
+import { CommonState } from 'audius-client/src/common/store'
+import { publishPlaylist } from 'audius-client/src/common/store/cache/collections/actions'
+import { getCollection } from 'audius-client/src/common/store/cache/collections/selectors'
+import { getUser } from 'audius-client/src/common/store/cache/users/selectors'
+// Importing directly from audius-client for now, this will be removed
+// when the profile page is implemented in RN
+import {
+  repostCollection,
+  undoRepostCollection,
+  saveCollection,
+  unsaveCollection,
+  shareCollection
+} from 'audius-client/src/common/store/social/collections/actions'
+import { open as openEditPlaylist } from 'audius-client/src/common/store/ui/createPlaylistModal/actions'
+import { setOpen as openDeletePlaylist } from 'audius-client/src/common/store/ui/delete-playlist-confirmation-modal/actions'
+import { getMobileOverflowModal } from 'audius-client/src/common/store/ui/mobile-overflow-menu/selectors'
+import {
+  OverflowAction,
+  OverflowActionCallbacks
+} from 'audius-client/src/common/store/ui/mobile-overflow-menu/types'
+import {
+  profilePage,
+  playlistPage,
+  albumPage
+} from 'audius-client/src/utils/route'
+import { push } from 'connected-react-router'
+
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+
+type Props = {
+  render: (callbacks: OverflowActionCallbacks) => React.ReactNode
+}
+
+const CollectionOverflowMenuDrawer = ({ render }: Props) => {
+  const dispatchWeb = useDispatchWeb()
+  const { id: modalId } = useSelectorWeb(getMobileOverflowModal)
+  const id = modalId as ID
+
+  const {
+    playlist_owner_id,
+    playlist_name,
+    is_album
+  } = useSelectorWeb((state: CommonState) => getCollection(state, { id: id }))
+
+  const { handle } = useSelectorWeb((state: CommonState) =>
+    getUser(state, { id: playlist_owner_id })
+  )
+
+  if (!id || !handle || !playlist_name || is_album === undefined) {
+    return <></>
```

Comment:
why fragment instead of null?

Reply:
Because I neglected the styleguide

 Fixed!